crawler/src/main/java/com/minibcr/crawler/Crawler.java

```diff
@@ -0,0 +1,78 @@
+package com.minibcr.crawler;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@EnableScheduling
+@PropertySource("classpath:crawler.properties")
+public class Crawler {
+
+    private static String URL;
+    private static String[] DISALLOWED;
+
+    private static YcombinatorPostBuilder ycombinatorPostBuilder = new YcombinatorPostBuilder();
+
+    @Scheduled(fixedRate = 30000)
+    public void run() {
+        Crawler crawler = new Crawler();
+
```

Comment:
Why are you creating an instance of Crawler inside the Crawler class?

Reply:
bc I'm silly. I moved that function into the crawler class from somewhere else and I didn't delete the instance. Good catch, thank you!